addon/components/sl-tab-panel.js

```diff
@@ -211,21 +208,28 @@ export default Ember.Component.extend({
     /**
      * The class determining how to align tabs
      *
```

Comment:
DocBlock should contain @throws {ember.assert} and contain a description of condition causing if it makes sense to do so